src/Tribe/Aggregator/Record/CSV.php

```diff
@@ -341,6 +341,14 @@ public function reset_tracking_options() {
 	}
 
 	public function continue_import() {
+		global $wpdb;
+		if ( empty( $wpdb->get_var( "SELECT GET_LOCK('tribe_ea_csv_import',10)" ) ) ) {
+			do_action( 'tribe_log', 'debug', 'CSV Import',
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 80b8ce2e795e1e42d5b43110c8383b62</sub>